### PR TITLE
Add SchemaExtension

### PR DESCRIPTION
**Pull request by sungam3r**
Support for http://spec.graphql.org/October2021/#SchemaExtension

### CODE REVIEW

**Comment by sungam3r** [comment]
Rel: https://github.com/graphql/graphql-spec/issues/921

File: .github/codecov.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
 # https://docs.codecov.com/docs/codecov-yaml
+# https://docs.codecov.com/docs/codecovyml-reference
+codecov:
+  require_ci_to_pass: false
```
Comment (sungam3r) [inline .github/codecov.yml:4]: May help 🤔

**Comment by sungam3r** [comment]
I merge this one and go on.

**Review by Shane32 — COMMENTED** [review]
Looks good to me.  (I must have missed this PR.)

**Pull request merged**